spec/acceptance/splunk_forwarder_spec.rb

```diff
@@ -5,6 +5,12 @@
     # Using puppet_apply as a helper
     it 'works idempotently with no errors' do
       pp = <<-EOS
+      class { '::splunk::params':
+        version      => '7.0.0',
+        build        => 'c8a78efdd40f',
+        src_root     => 'https://download.splunk.com',
+        splunkd_port => 8090,
```

Comment:
Ditto here, this should be mostly default (except port?)

Reply:
I had to do this because currently the acceptance test try to install both a splunk forwarder and a splunk enterprise instance on the same node.  Each product has a management utility that listens on port 8089 by default.  To get around this i just told the forwarder to be on the non-default port.

While this package does allow for uninstalling your splunk forwarder and splunk enterprise instance the uninstallation is technically not quite correct.  In order to prevent adding more scope to this already existing bug i did what i did above.

Reply:
But aren't version / build / src_root already set to those by default in params?
But if you're only setting the forwarder to a non-default port, can't `splunkd_port` be set in `splunk::forwarder` directly?

Reply:
I suppose.  that is the problem this module makes it confusing with its use of params.pp

I will:
1.) remove the defaults from params.pp
2.) move the splunkd_port to the forwarder class

Reply:
Done.